WooCommerce/Classes/POS/Utils/POSFontStyle.swift

```diff
@@ -34,8 +36,12 @@ enum POSFontStyle {
             Font.system(size: scaledValue(16, maximumContentSizeCategory: maximumContentSizeCategory), weight: .medium)
         case .posDetailEmphasized:
             Font.system(size: scaledValue(16, maximumContentSizeCategory: maximumContentSizeCategory), weight: .semibold)
-        case .posButtonSymbol:
-            Font.system(size: scaledValue(32, maximumContentSizeCategory: maximumContentSizeCategory), weight: .medium)
+        case .posButtonSymbolSmall:
```

Comment:
We should set the font size here not the pixel size of the parent container for the icon. That's why the icons appear too large.

<img width="120" alt="image" src="https://github.com/user-attachments/assets/2def563b-1546-490b-8256-9155e8f26c90" />

Reply:
Ah! Sorry for that! 🤦🤦 I'm just confirming that sizes are consistent in Slack and will give it a new pass with the inner sizes rather than the container ones.